src/test/ui/feature-gate/issue-43106-gating-of-builtin-attrs.rs

```diff
@@ -88,12 +93,18 @@
 #![crate_name = "0900"]
 #![crate_type = "bin"] // cannot pass "0800" here
 
-#![crate_id = "10"] //~ WARN use of deprecated attribute
+#![crate_id = "10"]
+//~^ WARN use of deprecated attribute
```

Comment:
I'm not sure what I changed that resulted in these popping up (but I think these changes seem correct?)